irods_consortium_continuous_integration_test_hook.py

```diff
@@ -15,18 +15,18 @@ def get_build_prerequisites_all():
 
```

Comment:
why not just return an empty list on line 13?

and leave the four lines below alone?